api/misc.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Check to see if the user requested a flag parameter.
+ *
+ * This returns one of three possible states:
+ * - null - no parameter with this key was set
+ * - true - parameter was set to "true" or an empty string (for ?flagname)
+ * - false - parameter was set to "false"
+ */
+function _get_flag_value($query_params, $flagname)
```

Comment:
Let's remove the initial `_` as that's suppose to be an indicator that it's only used in the file it was declared in.